### ISSUE TITLE
Change action on select generates non-functioning cypress code

### ISSUE THREAD

**Issue by cjhudson101** [cypress-io/cypress-chrome-recorder#45]
When clicking on and choosing a option within a select, the code generated on cypress export creates cy.get('element').type('value of option')

I believe when the element is of type select/option and the event is change we should do a cy.get('element').select('value or label of option selected') instead.

We simple select a value from a dropdown, export it, run the test in cypress and the select does not work properly.

**Comment by pmalhotra-eti** [comment]
@cjhudson101  -- Any lead or update on work around on this issue? This is turning out to be a small road-blocker for us.

**Comment by cjhudson101** [comment]
Nope.  We were just doing a POC with this thing for a hackathon and at the end of the two day event we decided this application is way too immature to bother with.

Most people at our organization have just written it off. :)

**Comment by germandupuy** [comment]
The same issue occurs with dropdown menus. Also, when navigating between pages in a grid. Has anyone found a solution or an answer?
